dockstore-yaml-proposals/jbrowse/stand_up.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Interesting. This `standup` script seems pretty general. I wonder whether the echos in the script will confuse developers. An invocation using the `dockstore.yml` could easily read the service name from the file for generating the start and stop messages.

For context, the init.d script http://shahmirj.com/blog/the-initd-script